pages/404.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import Card, { CardParagraph, CardTitle } from '../components/Card';
+import PrimaryLayout from '../components/PrimaryLayout';
+
+export default function NotFound() {
+  return (
+    <>
+      <PrimaryLayout />
+      <Card backgroundImageSource="/images/Volunteer.jpg">
+        <CardTitle>{'404 Page Not Found'}</CardTitle>
+        <CardParagraph>
+          {
+            ' The page you requested was not found. Always act in kindness by reporting errors like this at https://github.com/margaritahumanitarian/helpafamily/issues/new/choose'
```

Comment:
```suggestion
            'The page you requested was not found. Always act in kindness by reporting errors like this at https://github.com/margaritahumanitarian/helpafamily/issues/new/choose'
```

Reply:
Take a look at https://nextjs.org/docs/api-reference/next/link and see if you can implement a hyperlink using the `Link` component.

@ozer619 would you like to "Insert a suggestion" via the GitHub PR review tool that does this?